docs/cardano_lock.md

```diff
@@ -1,115 +1,154 @@
-# Cardano Lock
-
-## About cardano lock
-[Cardano](https://cardano.org/) is a blockchain platform for changemakers, innovators, and visionaries, with the tools and technologies required to create possibility for the many, as well as the few, and bring about positive global change.
+# [Cardano Lock](https://cardano.org)
```

Comment:
Add link in https://github.com/cryptape/ckb-auth/blob/main/README.md, pointed to this file.